eng/Generation.psm1

```diff
@@ -29,7 +29,7 @@ function Invoke-AutoRest($baseOutput, $projectName, $autoRestArguments, $sharedS
     {
         $outputPath = Join-Path $baseOutput "SomeFolder" "Generated"
     }
-    $namespace = $projectName.Replace('-', '_')
+    $namespace = $projectName.Replace('-', '_') -replace "(^|_)[a-z]",{return $_.Value.ToUpper()}
```

Comment:
Will this work on Windows as well?

Reply:
Yes, that is exactly to produce identical results on Windows and Linux